packages/amazonq/src/app/inline/completion.ts

```diff
@@ -203,6 +204,7 @@ export class AmazonQInlineCompletionItemProvider implements InlineCompletionItem
         context: InlineCompletionContext,
         token: CancellationToken
     ): Promise<InlineCompletionItem[] | InlineCompletionList> {
+        vsCodeState.isRecommendationsActive = true
```

Comment:
Does it make sense to wrap this in a try/finally to ensure it get appropriately reset in case of error.

Reply:
just updated to do this. We technically still need to set `vsCodeState.isRecommendationsActive = false` before the completion returns though but now if something fails it will always be reset